apps/front/src/translations/resources/fr.json

```
@@ -0,0 +1,71 @@
+{
+  "language": "Français",
+  "home": {
+    "play": {
+      "friend": "Jouer contre un ami",
+      "ai": "Jouer contre une IA"
+    },
+    "footer": "Le jeu original Knucklebones de Cult of the Lamb a été créé par Massive Monster.\nCe site est une création de fans et ne constitue pas une mise en œuvre officielle par Massive Monster.\nVous pouvez trouver le jeu original sur le site de <0>Cult of the Lamb</0>."
+  },
+  "guide": {
+    "label": "Comment jouer",
+    "goal": "Chaque tour, vous obtenez un dé aléatoire à placer dans une colonne.\nLe but est d'avoir plus de points que votre adversaire quand la partie se termine, c'est-à-dire quand l'un des 2 plateaux est plein.",
+    "rule1": "Vous obtenez encore plus de points quand le même dé apparaît plusieurs fois dans la même colonne.",
+    "rule2": "Vous retirez du plateau de votre adversaire tous les dés avec la même valeur en plaçant un dé dans la colonne correspondante de votre plateau."
+  },
+  "game-settings": {
+    "title": "Paramètres de la partie",
+    "difficulty": {
+      "label": "Difficulté de l'IA",
+      "easy": "Facile",
+      "medium": "Moyenne",
+      "hard": "Difficile"
+    },
+    "games": {
+      "label": "Nombre de rounds",
+      "indefinite": "Indéfini",
+      "1": "Best of 1",
+      "3": "Best of 3",
+      "5": "Best of 5"
+    },
+    "start": "Commencer la partie"
+  },
+  "menu": {
+    "style": {
+      "system": "Thème du système",
+      "dark": "Thème sombre",
+      "light": "Thème clair"
+    },
+    "share": {
+      "label": "Partager la partie",
+      "title": "Scannez le QR code pour inviter d'autres joueurs dans la partie",
+      "copy": "Copier le lien",
+      "copied": "Copié!"
+    },
+    "logs": {
+      "label": "Journaux",
+      "title": "Journaux"
+    },
+    "history": {
+      "label": "Round {{current}} sur {{boType}}",
+      "title": "Historique",
+      "empty-round": "Le premier round n'est pas encore terminé!",
+      "tie": "Égalité!",
+      "win": "{{player}} a gagné!",
+      "you-lead": "Vous avez l'avantage!",
+      "opponent-lead": "{{player}} a l'avantage!"
+    }
+  },
+  "game": {
+    "total": "Total",
+    "you": "Vous",
+    "ai": "IA",
+    "rematch": "Rejouer",
+    "continue": "Continuer",
+    "win-round": "{{player}} a gagné ce round avec {{points}} points !",
+    "win-game": "{{player}} a gagné avec {{points}} points !",
+    "waiting-rematch": "En attente de {{player}} ...",
```

```suggestion
    "waiting-rematch": "En attente de {{player}}...",
```
[Wikipedia](https://fr.wikipedia.org/wiki/Points_de_suspension#:~:text=ils%20ne%20sont%20jamais%20pr%C3%A9c%C3%A9d%C3%A9s%20d%E2%80%99une%20espace%2C%20sauf%20s%E2%80%99ils%20sont%20utilis%C3%A9s%20en%20remplacement%20d%E2%80%99un%20mot%20ou%20d%E2%80%99un%20groupe%20de%20mots%2C%20dans%20ce%20cas%20ils%20sont%20alors%20pr%C3%A9c%C3%A9d%C3%A9s%20et%20suivis%20de%20l%E2%80%99espacement%20normal%C2%A0%3B)